aries-site/src/pages/components/input.js

```diff
@@ -42,82 +42,113 @@ const Input = () => (
     </ContentSection>
     <ContentSection>
       <Subsection name="Checkbox">
-        <CheckboxExample />
-        <SubsectionText>
+      <SubsectionText>
           When the user needs to select one or more options, use a checkbox. The
           click target should include the checkbox label.
         </SubsectionText>
+        <Example
+          docs="https://v2.grommet.io/checkbox#props"
```

Comment:
```suggestion
          docs="https://v2.grommet.io/checkbox?theme=hpe#props"
```
For all of the props, we want to add this so that it loads grommet site in hpe theme (to match what was done for button I think)

Reply:
This is great thanks for suggesting! I didn't know that was a thing! lol